app/javascript/src/dashboard.js

```diff
@@ -116,24 +71,141 @@ $('document').ready(() => {
+  const editVolunteerPath = id => `/volunteers/${id}/edit`;
+  const casaCasePath = id => `/casa_cases/${id}`;
   var volunteersTable = $('table#volunteers').DataTable({
     autoWidth: false,
     stateSave: false,
-    columnDefs: [
+    columns: [
       {
-        targets: [1],
+        name: "display_name",
+        render: (data, type, row, meta) => {
+          return `
+            <a href="${editVolunteerPath(row.id)}">
+              ${row.display_name || row.email}
+              ${row.made_contact_with_all_cases_in_days === 'false'
+                ? `🕐 <i class="fa fa-question-circle" aria-hidden="true" data-toggle="tooltip" title="Has at least one case with no contact in 14 days"></i>`
+                : ''
+              }
+            </a>
+          `;
+        }
+      },
+      {
+        name: "email",
+        render: (data, type, row, meta) => row.email,
         visible: false
       },
       {
-        targets: [6],
+        className: "supervisor-column",
+        name: "supervisor_name",
+        render: (data, type, row, meta) => {
+          if (!row.supervisor.id) return "";
+
+          return `
+            <a href="${editSupervisorPath(row.supervisor.id)}">
+              ${row.supervisor.name}
+            </a>
+          `;
+        }
+      },
+      {
+        name: "active",
+        render: (data, type, row, meta) => row.active === "true" ? "Active" : "Inactive",
+        searchable: false,
+      },
+      {
+        name: "has_transition_aged_youth_cases",
+        render: (data, type, row, meta) => row.has_transition_aged_youth_cases === "true" ? "Yes 🐛🦋" : "No",
+        searchable: false
+      },
+      {
+        name: "casa_cases",
+        render: (data, type, row, meta) => {
+          const links = row.casa_cases.map(casaCase => {
+            return `<a href="${casaCasePath(casaCase.id)}">${casaCase.case_number}</a>`;
+          });
+
+          return links.join(", ");
+        },
+        orderable: false
+      },
+      {
+        name: "most_recent_contact_occurred_at",
+        render: (data, type, row, meta) => {
+          if (!row.most_recent_contact.case_id) return "None ❌";
+
+          return `
+            <a href="${casaCasePath(row.most_recent_contact.case_id)}">
+              ${row.most_recent_contact.occurred_at}
+            </a>
+          `;
+        },
+        searchable: false,
         visible: false
       },
       {
-        targets: [7],
+        name: "contacts_made_in_past_days",
+        render: (data, type, row, meta) => row.contacts_made_in_past_days,
+        searchable: false,
         visible: false
+      },
+      {
+        name: "actions",
+        orderable: false,
+        render: (data, type, row, meta) => {
+          return `
+            <a href="${editVolunteerPath(row.id)}">
+              Edit
+            </a>
+          `
+        },
+        searchable: false
       }
-    ]
+    ],
+    processing: true,
```

Comment:
We need capybara test coverage for this or I worry that it will break the next time we update datatables